modules/mono/editor/bindings_generator.h

```diff
@@ -614,8 +623,10 @@ class BindingsGenerator {
 	HashMap<const MethodInterface *, const InternalCall *> method_icalls_map;
 
 	HashMap<StringName, List<StringName>> blacklisted_methods;
+	HashSet<StringName> compat_singletons;
 
 	void _initialize_blacklisted_methods();
+	void _initialize_compat_singletons();
```

Comment:
This is never called yet, even tho the method is empty right now it should already be called from the appropriate place in `_initialize`

Reply:
Sorry, I forgot to add it. Fixed.